CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ This is a changelog for Piwik platform developers. All changes for our HTTP API'
 ### Internal Change
 * The option `branch` of the console command `development:sync-system-test-processed` was removed as it is no longer needed.
 
+### APIs
+* There is a new event `CronArchive.getIdSitesNotUsingTracker` that allows you to set a list of idSites that do not use the Tracker API to make sure we archive these sites if needed.
```

Comment:
> Maybe getIdSitesNotUsingTrackerAPI would be clearer? 

In general, the issue I have w/ these names is that it's not clear what the intent is. The event is used to force CronArchive to send API requests to initiate archiving, but `getIdSitesNotUsingTrackerAPI` doesn't convey that information. I think it might be very easy for a plugin developer to overlook the event entirely. Maybe `CronArchive.getIdSitesToForceArchivingRequestsFor`? Or if you think `getIdSitesNotUsingTrackerAPI` might be re-used in a different context, it would be ok.

> Do you have an idea for that event name? I could maybe implement it

How about `Archiver.getIdSitesDisplayingThirdPartyData` or `Archver.getIdSitesDisplayingNonPiwikData`? This isn't needed for this PR, but I think it would be a good stop-gap measure for 2.15. Since 3.0 won't be released for say at least 6 months or so, this event would help plugin developers in the meantime. What do you think? If you agree, I'll create a new issue.